node-fs-readfile/read-any.ts

```diff
@@ -0,0 +1,8 @@
+import { readFile } from 'node:fs/promises';
+const arg = process.argv[2];
+try {
+  const message = await readFile('./' + arg, { encoding: 'utf8' });
```

Comment:
the `./` is fine but not necessary. When we are importing files in Node it is necessary but not required for this method. 

`{ encoding: 'utf8' }` could just be `utf8`. If you are passing in multiple options then the object with the property is required. What you did was fine but just wanted to point out there is a shorter way to set the encoding.